deep_qa/training/optimizers.py

```diff
@@ -14,18 +14,27 @@
 \* I should also note that Keras is an incredibly useful library that does a lot
 of things really well. It just has a few quirks...
 """
+import logging
 from typing import Union
-from keras.optimizers import SGD, RMSprop, Adagrad, Adadelta, Adam, Adamax, Nadam
+
+# pylint: disable=no-name-in-module
```

Comment:
I think you can get around this by adding tensorflow here: https://github.com/allenai/deep_qa/blob/82534a7d6340ee986e3c9be28f30f8eb11cf8a19/.pylintrc#L35